src/js/rx/components/GlossaryList.jsx

```diff
@@ -9,21 +9,32 @@ class GlossaryList extends React.Component {
 
     Object.keys(terms).forEach(o => {
       const id = terms[o].term.toLowerCase().replace(/\s/g, '');
-      termsList.push(<dt id={id} key={++tKey}>{terms[o].term}</dt>);
-      termsList.push(<dd key={++tKey}>{terms[o].definition}</dd>);
+      termsList.push(<p className="glossary-title" id={id} key={++tKey}>{terms[o].term}</p>);
+      termsList.push(<p className="glossary-body" key={++tKey}>{terms[o].definition}</p>);
     });
 
     let title;
     if (this.props.title) {
       title = <h2 className="rx-pgroup-title va-h-ruled">{this.props.title}</h2>;
     }
 
+    // check if the array has length 1, then render a certain way
+    // else render the way it is
+
+    if (termsList.length === 2) {
+      return (
+        <div>
+          <h3 className="modalTitle">Glossary: {termsList[0]}</h3>
+          {termsList[1]}
+        </div>
+      );
+    }
     return (
```

Comment:
Just to confirm, there's no scenario that `termsList` could wind up as an empty array, right?

Reply:
Nope, none. It's being fed by a predefined list.